docs/pages/modules/additional-platform-support.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Additional platform support
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
```

Comment:
```suggestion
Expo Modules API provides first-class support for Android and iOS. However, since all Apple platforms are based on the same foundation and use the same programming language, targeting other [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms) in the Expo module is possible. At the moment, only **macOS** and **tvOS** platforms are supported.
This guide will walk you through adding support for these platforms.
```